rebasehelper/application.py

```diff
@@ -678,11 +680,27 @@ def get_rebased_patches(self):
             output_patch_string.append('Patches were not touched. All were applied properly')
         return output_patch_string
 
-    def print_summary(self):
-        output = output_tool.OutputTool(self.conf.outputtool)
-        report_file = os.path.join(self.results_dir, self.conf.outputtool + settings.REBASE_HELPER_OUTPUT_SUFFIX)
-        output.print_information(path=report_file)
-        logger.info('\nReport file from rebase-helper is available here: %s', report_file)
+    def print_summary(self, exception=None):
+        """
+        Save rebase-helper result and print the summary using output_tools_runner
+        :param exception: Error message from rebase-helper
+        :return:
+        """
+        log = None
+        # Store rebase helper result exception
+        if exception:
+            if exception.logfile is not None:
+                if os.path.exists(exception.logfile):
```

Comment:
This could be written as oneliner: `if exception.logfile and os.path.exists(exception.logfile):`. But if you prefer to keep it on two lines, I'm okay with that.